cypress/integration/onboarding/email/existing_email_adress_spec.js

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Email Address', () => {
+    before(() => {
+        // 1. Go to / and logout
+        cy.visit('/');
+        cy.apiLogout();
+    });
+
+    it('M16363 Should not create account with an existing email address', () => {
+    // Create a new user
+        // 2. Go to /login
+        cy.visit('/login');
+
+        // 3. Click on sign up button
+        cy.get('#signup').click();
+
+        // 4. Type 'unique.email@sample.mattermost.com' for email
+        cy.get('#email').type('unique.email@sample.mattermost.com');
+
+        // 5. Type 'unique-1' for username
+        cy.get('#name').type('unique-1');
+
+        // 6. Type 'unique1pw' for password
+        cy.get('#password').type('unique1pw');
+
+        // 7. Click on Create Account button
+        cy.get('#createAccountButton').click();
+
+        // * Verify there is Logout Button
+        cy.contains('Logout').should('be.visible');
+
+        // * Verify 'Teams you can join' is visible
+        cy.contains('Teams you can join').should('be.visible');
```

Comment:
While this works perfectly fine for now. This can break in the event of changes to the text which triggers false alarms. Please feel free to create unique ID's to select `Teams you can join` container so you can do something like
`cy.get('#teamsYouCanJoinContainer').assertion`. 

We don't usually change Id's in HTML and if we do change we usually check them across the app to be sure that we don't break any existing functionalities and tests making this more obvious.

Same case with other instances like `div.form-group.has-error`. Feel free to add ID's for selecting elements

Reply:
I agree, using IDs instead of something that can change makes more sense. Thank you for suggesting!